tests/CMakeLists.txt

```diff
@@ -306,6 +306,7 @@ add_custom_command(OUTPUT test-app-login
         ../Schema/authn/001-initial.blue.sql
         ../Schema/app/002-initial.blue.sql
         ../Configuration/odin-views.json
+        ../Static/app/login.html
```

Comment:
I don't know about this. In a sense, yes it does depend on this, but in another sense, we aren't going to re-run the test just because the HTML page changes are we? The functionality depends on this file, but I don't think that the test does.